src/functions/InputWidth.js

```diff
@@ -1,8 +1,8 @@
 const InputWidth = (text) =>{
```

Comment:
Please don't limit your input symbols, let it be and set your task container stretchable, set min-height